src/components/vm/disks/diskAdd.jsx

```diff
@@ -149,37 +151,39 @@ class AdditionalOptions extends React.Component {
 
         return (
             <ExpandableSection toggleText={ this.state.expanded ? _("Hide additional options") : _("Show additional options")}
-                               onToggle={() => this.setState({ expanded: !this.state.expanded })} isExpanded={this.state.expanded} className="add-disk-additional-options">
-                <Grid hasGutter md={6}>
-                    <FormGroup fieldId='cache-mode' label={_("Cache")}>
-                        <FormSelect id='cache-mode'
-                            onChange={value => this.props.onValueChanged('cacheMode', value)}
-                            value={this.props.cacheMode}>
-                            {diskCacheModes.map(cacheMode => {
-                                return (
-                                    <FormSelectOption value={cacheMode} key={cacheMode}
-                                                      label={cacheMode} />
-                                );
-                            })}
-                        </FormSelect>
-                    </FormGroup>
-
-                    <FormGroup fieldId={this.props.idPrefix + '-bus-type'} label={_("Bus")}>
-                        <FormSelect id={this.props.idPrefix + '-bus-type'}
-                            data-value={this.props.busType}
-                            onChange={value => this.props.onValueChanged('busType', value)}
-                            value={this.props.busType}>
-                            {displayBusTypes.map(busType => {
-                                return (
-                                    <FormSelectOption value={busType.value}
-                                                      key={busType.value}
-                                                      isDisabled={busType.disabled}
-                                                      label={busType.value} />
-                                );
-                            })}
-                        </FormSelect>
-                    </FormGroup>
-                </Grid>
+                               onToggle={() => this.setState({ expanded: !this.state.expanded })} isExpanded={this.state.expanded} className="pf-u-pt-md">
+                <Form onSubmit={e => e.preventDefault()} isHorizontal>
+                    <Grid hasGutter md={6}>
+                        <FormGroup fieldId='cache-mode' label={_("Cache")}>
+                            <FormSelect id='cache-mode'
+                                onChange={value => this.props.onValueChanged('cacheMode', value)}
+                                value={this.props.cacheMode}>
+                                {diskCacheModes.map(cacheMode => {
+                                    return (
+                                        <FormSelectOption value={cacheMode} key={cacheMode}
+                                                          label={cacheMode} />
+                                    );
```

Comment:
Please avoid introducing imperative code blocks when you just return a value. A clean `k => v` mapping is functional and nicer than `k => { return v }`. I know the code was just moved, but let's clean it up while we are touching it.

Reply:
If not, you can also do `=> ( ...` and then enclose a multi-line value. However, for a single JSX element you can just do `k => <Element ...>` and add multiple properties on the next line (with the usual alignment).

Reply:
Sure, that's fine.